src/client.ts

```diff
@@ -2305,6 +2305,28 @@ export class StreamChat<
     );
   }
 
+  /** deactivateAndDeleteUser - convenience method that both deactivates and deletes a user
+   *
+   * @param {string} userID target user ID
+   * @param {object} [options] options for both deactivateUser and deleteUser
+   * @returns {Promise<APIResponse>}
+   */
+  async deactivateAndDeleteUser(
+    userID: string,
+    options?: {
```

Comment:
It is better to extract these options to types and combine here. Otherwise easy to go outdated.